### ISSUE TITLE
.neighbors() call results in "out of memory" error

### ISSUE THREAD

**Issue by danburzo** [mourner/flatbush#50]
Hi @mourner, I hope you are well!

Upgrading to the latest flatbush version from `3.3.0` produced a regression in `.neighbors()` when `k = Infinity`, resulting in an 'out of memory' error: 

```js
const Flatbush = require('flatbush@3.3.1');
const bush = new Flatbush(1);
bush.add(1,2,3,4);
bush.finish();
bush.neighbors(1, 2);
``` 

It's related to the change in https://github.com/mourner/flatbush/commit/6903d4b2c430628dbaaac19b11a1d12da97b285e but I didn't trace the root cause — please let me know if I should look into that.

**Comment by mourner** [comment]
Yeah, I already fixed it in https://github.com/mourner/flatbush/commit/f3d24b3ba3e67abcb9a531dfc8095a5922ca83c0 but haven't pushed a new release yet. Will do tomorrow 👍

**Comment by danburzo** [comment]
Ah, I missed that commit. Awesome!

**Comment by mourner** [comment]
@danburzo one caveat is that you'd have to upgrade to v4+ which ditches the CommonJS entry point, so the example above wouldn't work because of `require`.

**Comment by mourner** [comment]
Released in v4.2.0.

**Comment by danburzo** [comment]
Thank you!
